### PR TITLE
remove the PartialApp analyzer

### LINKED ISSUE #68: Remove PartialAppAnalyzer
@Smaug123 Would you be okay with us removing the PartialAppAnalyzer?
It's pretty much disabled everywhere and given recent events, the need to have it seems to be gone.

### PR DESCRIPTION
**Pull request by dawedawe**
fixes #68

### CODE REVIEW

**Review by Smaug123 — DISMISSED** [review]

**Comment by nojaf** [comment]
Could you add a changelog entry? `### Removed` section

**Comment by dawedawe** [comment]
> Could you add a changelog entry? `### Removed` section

Oops, sorry.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [Unreleased]
+
+### Removed
+
+* Removed the PartialAppAnalyzer
```
Comment (nojaf) [inline CHANGELOG.md:7]:
```suggestion
* Removed the PartialAppAnalyzer. [#68](https://github.com/G-Research/fsharp-analyzers/issues/68)
```

**Review by nojaf — APPROVED** [review]
Thanks

**Pull request merged**